dhctl/pkg/kubernetes/actions/deckhouse/log.go

```diff
@@ -82,15 +85,43 @@ func isErrorLine(line *logLine) bool {
 		return true
 	}
 
+	// Consider stderr messages are errors too.
 	if line.Output == "stderr" {
-		// skip tiller output
-		if line.Component == "tiller" {
-			return false
+		// Skip possible messages from tiller.
+		if line.Component != "tiller" {
+			return true
 		}
+	}
+
+	return false
+}
 
-		return false
+// isModuleSuccess returns true on message about successful module run.
+func isModuleSuccess(line *logLine) bool {
+	// Message about successful ModuleRun since PR#126 in flant/addon-operator.
+	// https://github.com/flant/addon-operator/blob/7e814fbe92fb12af79c67c4226b4c2781d959f3c/pkg/addon-operator/operator.go#L1376
+	if line.Message == "ModuleRun success, module is ready" {
+		return true
 	}
+	// Message about successful ModuleRun prior PR#126 in flant/addon-operator.
+	if line.Message == "Module run success" {
+		return true
+	}
```

Comment:
This message not in deckhouse since May 2020. May be it is time to remove this condition?

Reply:
why not